client/taskenv/env_test.go

```diff
@@ -244,10 +243,10 @@ func TestEnvironment_AsList_Old(t *testing.T) {
 			Networks: []*structs.NetworkResource{
 				{
 					Device:        "eth0",
-					IP:            "192.168.0.100",
-					ReservedPorts: []structs.Port{{Label: "admin", Value: 5000}},
+					IP:            "127.0.0.1",
+					ReservedPorts: []structs.Port{{Label: "https", Value: 8080}},
 					MBits:         50,
-					DynamicPorts:  []structs.Port{{Label: "http", Value: 2000}},
```

Comment:
I'm unclear how values in the tests were chosen - given that rest of tests don't reference `admin` port and 2000/5000 don't appear in the final expected results.  I did my best to capture what the intent of test is and made values consistent with how this and the values in `TestEnvironment_AsList` so both tests get the same expected output.